man/bst.1.scd

```diff
@@ -350,6 +350,75 @@ Supported interface types are:
 	- *l3*
 	- *l3s*
 
+# EXAMPLES
+
+## Using bst to get an isolated shell
+
+The default invocation of bst will provide a shell as isolated as possible on
+your host system.
+
+*$ id*++
+uid=1000(barney) gid=1000(barney) groups=1000(barney)++
+*$ bst*
+
+This shell sees all of its namespaces unshared. Most notably, you are root
+inside the spacetime, by virtue of the user namespace being unshared:
+
+*# id*++
+uid=0(root) gid=0(root) groups=0(root)
+
```

Comment:
I would add something like this:

and you have full (root-style) control over your namespaces:

```
   ls /tmp
   <see lots of suff>
   mount -t tmpfs none /tmp && echo hello > /tmp/greeting
   ls /tmp
   <see only greeting>

```
because it's just so cool to have the ability to perform these sorts of manipulations in isolation of other users.